src/Alerts/ToastNotificationWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+/**
+ * A wrapper for the ToastNotification component to position them at the top
+ * right of the page.
+ */
+const ToastNotificationWrapper = styled.div`
+  display: flex;
+  align-self: flex-end;
+  flex-direction: column;
+  height: 100%;
+  overflow-y: auto;
+  position: absolute;
+  top: 5px;
+  right: 5px;
+`;
```

Comment:
```suggestion
z-index: 1000;
`;
```
I think we also need to set a high z-index for accessibility and usability (to ensure it remains on top of other content)

Reply:
Good point. This was updated in commit [99f6377](https://github.com/seas-computing/mark-one/pull/198/commits/99f6377d6206e410518ae2e0f048ef8e09ea07e0).